scss/reset/_bodyText.scss

```diff
@@ -263,14 +269,13 @@ in order to also set the color of icons inline with text.
 </section>
 ```
 */
-@each $colorName, $color in (
-	'red': $C_red,
-	'pink': $C_pink,
-	'teal': $C_teal,
-	'blue': $C_blue,
-	'yellow': $C_yellow
-) {
-	@include _modifier('text', "#{$colorName}") {
+@each $colorName,
+	$color in ("red": $C_red,
```

Comment:
can we break lines for all the list properties in the `()` here? It's nice that stylelint is breaking `each` arguments, but the colors list looks weird... just wondering if we can further break lines here for readability without the lint task reformatting